mule-extensions-api/src/main/java/org/mule/runtime/extension/api/util/NameUtils.java

```diff
@@ -289,13 +290,35 @@ public static String defaultNamespace(String extensionName) {
       throw new IllegalArgumentException("No Namespace can be created from an empty name");
     }
 
-    String namespace = deleteWhitespace(extensionName);
+    String namespace = extensionName.trim();
     namespace = removeEndIgnoreCase(namespace, "extension");
     namespace = removeEndIgnoreCase(namespace, "connector");
     namespace = removeEndIgnoreCase(namespace, "module");
+
+    if (namespace.contains(SPACE)) {
```

Comment:
Maybe we should decide if this is what has to be changed or the name this is being compared to. Anyway the two names have to be unified. We are reviewing this.